miio/waterheater.py

```diff
@@ -0,0 +1,133 @@
+import logging
+from typing import Any, Dict
+
+from .click_common import command, format_output
+from .device import Device
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class WaterHeaterStatus:
+
+    def __init__(self, data: Dict[str, Any]) -> None:
+        self.data = data
+
+    @property
+    def washStatus(self) -> str:
```

Comment:
```suggestion
    def status(self) -> str:
```
Which statuses are known? Could it be exposed as an enum?